Data-UB/AddOns/Data-UB-Vanilla/Ja2_Options.INI

```diff
@@ -90,13 +92,13 @@ MAX_NUMBER_PLAYER_VEHICLES = 2
 ; Use these to adjust the numbers of various sorts of "people" that can appear in the same sector in TACTICAL mode.
 ;------------------------------------------------------------------------------------------------------------------------------
 
-; Enemies (i.e., soldiers), valid values 16 through 64, default is 32
+; Enemies (i.e., soldiers), valid values 16 through 256, default is 32
 MAX_NUMBER_ENEMIES_IN_TACTICAL = 20

```

Comment:
comment says "default is 32" but value is 20.

suggest just removing this "default is" part of the comment and letting the value speak for itself.

same for wherever this repeats